src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -384,8 +383,8 @@ public Mono<Void> deleteModificationGroup(UUID groupUuid) {
 
     public List<ModificationInfos> doAction(NetworkStoreListener listener, Runnable action,
                                             NetworkModificationException.Type typeIfError,
-                                            UUID networkUuid, ReporterModel reporter,
-                                            Reporter subReporter) {
+                                            UUID reportUuid, Supplier<ReporterModel> reporter,
```

Comment:
why change from reporter to Supplier<Reporter> ?

Reply:
Removed

Reply:
Delete also for subReporter ?

Reply:
Only do AtomicReference.get

Reply:
Done